source/scheme/mus_scheme_derived_quantities_type_module.f90

```diff
@@ -202,7 +202,8 @@ function mus_assign_derived_functions_ptr(label_stencil, label_fluid) result(get
     getQuantities%momentum_from_vel_dens_ptr => get_momentum_from_vel_dens_incompressible
     getQuantities%kineticEnergy_from_vel_dens_ptr => get_kineticEnergy_from_vel_dens_incompressible
     getQuantities%rho0Inv_ptr => get_rho0Inv_incompressible
-  else 
+  else
+    !! TODO: add pointers to obtain derived quantities for other fluid types
```

Comment:
Todo other fluid types

Reply:
Suggestion:
1. Replace the abort with a warning that "derived function pointer is not implemented for the scheme kind"
2. Rename label_fluid to scheme kind
3. Replace if else with select case on scheme kind.